packages/components/src/palette-edit/test/index.tsx

```diff
@@ -69,7 +82,6 @@ describe( 'getNameForPosition', () => {
 
 describe( 'PaletteEdit', () => {
 	const defaultProps = {
-		gradients: false,
```

Comment:
What do you think about disallowing a `gradients` prop for `<PaletteEdit>` if there's a colors prop?

That's how [PaletteEdit is used in Gutenberg](https://github.com/search?q=repo%3AWordPress%2Fgutenberg%20%3CPaletteEdit&type=code).

It makes the [discriminated union](https://github.com/WordPress/gutenberg/pull/47764/files#diff-532dfa00a13a98be8d7f9bb3569fd9437ffa5e9988d7a7cc5031ff28d659cf46R57-R85) easier to understand.

Reply:
Sounds good. For what it's worth, I don't consider this a breaking change, especially since it is just a type check 🙂